build/tasks/spec-task.coffee

```diff
@@ -106,27 +112,37 @@ module.exports = (grunt) ->
 
       callback(null, error)
 
-  grunt.registerTask 'run-specs', 'Run the specs', ->
+  grunt.registerTask 'run-specs', 'Run the specs', (packageName) ->
     done = @async()
     startTime = Date.now()
 
-    # TODO: This should really be parallel on both platforms, however our
-    # fixtures step on each others toes currently.
-    if process.platform in ['darwin', 'linux']
-      method = async.parallel
-    else if process.platform is 'win32'
-      method = async.series
+    if packageName?
+      runSinglePackageSpecs packageName, (error) ->
+        elapsedTime = Math.round((Date.now() - startTime) / 100) / 10
+        grunt.log.ok("Total spec time: #{elapsedTime}s using #{concurrency} cores")
+        grunt.log.error("[Error]".red + " #{packageName} spec(s) failed") if error?
+        if process.platform is 'win32' and process.env.JANKY_SHA1
+          done()
+        else
+          done(!error?)
```

Comment:
Hmm, I thought `done(error)` would work here, does it need to be coerced into a boolean?